R/R/model.R

```diff
@@ -42,6 +42,8 @@
 #' "unconstrained". By default, if intercept is negative, Robyn will drop intercept
 #' and refit the model. Consider changing intercept_sign to "unconstrained" when
 #' there are \code{context_vars} with large positive values.
+#' @param rssd_0eff Logical. When TRUE, the objective function DECOMP.RSSD will
```

Comment:
- then rssd_zero_penalty 
- ok
- No, it's part of the objective function, not just a score for output. or where do you suggest?
- It's actually quite clear IMO. Given the same RSSD score, a model with 50% 0-coef will get penalized by RSSD * 1.5 (larger error), while another model with no 0-coef gets unpenalized with RSSD *1
- For refresh, I'm not sure at the moment, because the RSSD for refresh is calculated from the net new refresh datapoints, with the purpose to have beta coefs reflecting bigger spend changes --> If I stop spending on channel A for a week, ideally this should be reflected or not penalized. I'm not sure about this yet.
- Yes we should!

Reply:
Ok, pending only:
- Update site with this new parameter information (even though I've added your example to the internal parameter documentation).
- Agree on refresh default value.

QQ: why would a user set `rssd_zero_penalty = FALSE`? Is it necesary we enable it as a parameter?